backend/src/graphql/resolver/AdminResolver.test.ts

```diff
@@ -261,6 +263,216 @@ describe('AdminResolver', () => {
+        })
+
+        it('returns an error', async () => {
+          await expect(
+            query({
+              query: searchUsers,
+              variables: {
+                ...variablesWithoutTextAndFilters,
+              },
+            }),
+          ).resolves.toEqual(
+            expect.objectContaining({
+              errors: [new GraphQLError('401 Unauthorized')],
+            }),
+          )
+        })
+      })
+
+      describe('with admin rights', () => {
+        beforeAll(async () => {
+          admin = await userFactory(testEnv, peterLustig)
+          await query({
+            query: login,
+            variables: { email: 'peter@lustig.de', password: 'Aa12345_' },
+          })
+
+          await userFactory(testEnv, bibiBloxberg)
+          await userFactory(testEnv, stephenHawking)
+          await userFactory(testEnv, garrickOllivander)
+        })
+
+        afterAll(async () => {
+          await cleanDB()
+          resetToken()
+        })
+
+        describe('find', () => {
+          const allUsers = {
+            bibi: expect.objectContaining({
+              email: 'bibi@bloxberg.de',
+            }),
+            garrick: expect.objectContaining({
+              email: 'garrick@ollivander.com',
+            }),
+            peter: expect.objectContaining({
+              email: 'peter@lustig.de',
+            }),
+            stephen: expect.objectContaining({
+              email: 'stephen@hawking.uk',
+            }),
+          }
+
+          it('all users by "filters === null"', async () => {
+            await expect(
+              query({
+                query: searchUsers,
+                variables: {
+                  ...variablesWithoutTextAndFilters,
+                },
+              }),
+            ).resolves.toEqual(
+              expect.objectContaining({
+                data: {
+                  searchUsers: {
+                    userCount: 4,
+                    userList: expect.arrayContaining(convertObjValuesToArray(allUsers)),
+                  },
+                },
+              }),
+            )
+          })
+
+          it('all users by "filterByActivated === null && filterByDeleted === null"', async () => {
+            await expect(
+              query({
+                query: searchUsers,
+                variables: {
+                  ...variablesWithoutTextAndFilters,
+                  filters: {
+                    filterByActivated: null,
+                    filterByDeleted: null,
+                  },
+                },
+              }),
+            ).resolves.toEqual(
+              expect.objectContaining({
+                data: {
+                  searchUsers: {
+                    userCount: 4,
+                    userList: expect.arrayContaining(convertObjValuesToArray(allUsers)),
+                  },
+                },
+              }),
+            )
+          })
+
+          it('users with unchecked email', async () => {
```

Comment:
New describe block.
`filter by unchecked email`

```suggestion
          it('finds only users with unchecked email', async () => {
```

Reply:
Cool! 😍
Done.